digitalocean_k8s_ubuntu_16.04_node.sh

```diff
@@ -5,26 +5,42 @@
 # ------------------------------------------------------------------------------------------------------------------------
 
 # Specify the Kubernetes version to use.
-KUBERNETES_VERSION="1.9.2"
+KUBERNETES_VERSION="1.11.0"
```

Comment:
Same as above. If we decide to bump up Kube version, don't forget to update here as well.